tljh/schemas/tljh_configurator.py

```diff
@@ -0,0 +1,27 @@
+from jupyterhub_configurator.hooks import hookimpl
+
+@hookimpl
+def jupyterhub_configurator_fields():
```

Comment:
Let's put this file in https://github.com/yuvipanda/jupyterhub-configurator for now? The format of these schemas is changing pretty rapidly. We can refer to this from there!